src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -3,6 +3,7 @@
  */
 package net.snowflake.client.jdbc;
 
+import static net.snowflake.client.core.QueryStatus.NO_DATA;
 import static net.snowflake.client.core.QueryStatus.RUNNING;
```

Comment:
remove these lines and use `QueryStatus` to make consistent with other constant variable usage.

Reply:
Sure, will fix them!